### PR TITLE
766 make submitWizard not stateful

### PR DESCRIPTION
**Pull request by mrblippy**
#766

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java [hunk 1]
```diff
@@ -768,6 +771,7 @@ public void fireDirect(SectionId sectionId, SectionInfo info2) throws Exception
                 powerPage.saveToDocument(info2);
                 PropBagEx newXml = powerPage.getDocBag();
                 setXml(newXml.toString());
+                // setSubmitWizard(false);
```
Comment (doolse) [inline Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java]: No need for this

File: Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java [hunk 2]
```diff
@@ -661,10 +661,13 @@ public QueryCallback(AjaxRenderContext context, AjaxUpdateData reloadData) {
...
   public class SearchQueryModel extends AbstractResetFiltersQuerySection.AbstractQuerySectionModel {
-    @Bookmarked(name = "sw")
+    @Bookmarked(name = "sw", stateful = false)
     private boolean submitWizard;
 
-    @Bookmarked(name = "editquery", parameter = "editquery", supported = true)
+    @Bookmarked(
+        name = "editquery",
+        parameter = "editquery",
+        supported = true) // , contexts = BookmarkEvent.CONTEXT_SESSION)
```
Comment (doolse) [inline Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java]: No need to change this

File: Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java (hunk 1 above)
Comment (mrblippy) [reply Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java]: Oops, forgot that was there

File: Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java (hunk 2 above)
Comment (mrblippy) [reply Source/Plugins/Core/com.equella.core/src/com/tle/web/searching/section/SearchQuerySection.java]: Oops, forgot that was there

**Pull request merged**